styles/diary.css

```diff
@@ -0,0 +1,167 @@
+/* Diary containers */
+
+
+.diary-section {
+    display: flex;
+    flex-wrap: wrap;
+    /* background-color: #DCA47C; */
+    width: 1160px;
+    margin: 50px 30px;
+}
+
+.breakfast, .dinner {
+    background-color: #698474;
+    border-radius: 10px;
+}
+
+
+.diary-content {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    height: 250px;
+    width: 100%;
+    margin-top: 20px;
+    margin-bottom: 40px;
+}
+
+.diary-heading {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 50px;
+    padding-top: 10px;
+}
+
+.diary-heading * {
+    font-size: xx-large;
+    font-family: "Roboto Slab", serif;
+    font-optical-sizing: auto;
+    font-weight: 500;
+    font-style: normal;
+    margin: 0px 50px;
+}
+
+.diary-meal-container {
+    display: flex;
+    width: 100%;
+    margin-top: 20px;
+}
+
+.diary-entry-recipe {
+    width: 30%;
+}
+.diary-entry-data {
+    width: 70%;
+}
+
+/* Recipe blocks */
+
+.recipe-container {
+    background-color: #FFD3B6;
+    height: 250px;
+    width: 250px;
+    font-size: x-large;
+    border-radius: 10px;
+    margin: 20px;
+}
+
+.rec-img {
+    /* background-color: #DCA47C; */
+    background-color: #DCA47C;
+    border-top-left-radius: 10px;
+    border-top-right-radius: 10px;
+    height: 50%;
+    align-items: flex-start;
+}
+
+.fodmap-index-label {
+    width: 60%;
+    text-align: center;
+}
+
+.low-fodmap {
+    background-color: lightblue;
+}
+
+.high-fodmap {
+    background-color: lightcoral;
+}
+
+.rec-details * {
+    font-family: "Roboto Slab", serif;
+    font-size: medium;
+    font-weight: 300;
+    margin: 8px 10px;
+}
+
+.rec-title {
+    font-family: "Roboto Slab", serif;
+    font-size: larger;
+    font-weight: 500;
+}
+
+.left-arrow, .right-arrow {
```

Comment:
You can call it just arrow

.arrow {
    background-color: #698474;
    width: 45px;
    height: 40px;
    text-align: center;
    border-radius: 15px;
    color: #FCF8F3;
    font-weight: bolder;
}
and if you will need some specific code for left and right then:

.left-arrow {}
.right-arrow {}

Reply:
I thought that I might need to have separate classes for each side but this was not needed in the end. I have called them both arrow now.